src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -33,6 +34,34 @@
 
 #define INCLUDE_PROCESSOR_PROXY_FAILURE 44
 
+DEFINE_bool(local_fallback, true, "Fallback to local execution if remote "
+            "execution fails");
+DEFINE_bool(verbose, false, "Verbose mode");
+DEFINE_bool(force_remote, false, "Force the command to be run remotely");
+DEFINE_bool(accept_cached, true, "Indicate whether to accept action cache results");
+DEFINE_bool(save_exec_data, false, "Indicate whether to save full execution data");
+DEFINE_bool(allow_out_under_in, false, "Indicate whether to allow outputs under"
+            " input directories");
+DEFINE_bool(allow_out_dirs, false, "Indicate whether to allow output directories"
+            " as inputs");
+DEFINE_string(out_dir, "out", "Name of the output directory");
+DEFINE_string(invocation_id, "", "Invocation ID of the build");
+DEFINE_int32(retry_count, 1, "Number of remote retries to attempt");
+DEFINE_int32(proxy_instances, 1, "Number of remote client proxy instances");
+DEFINE_string(proxy_address, "localhost:8080", "Address of the remote client proxy");
+DEFINE_int32(server_instances, 1, "Number of include processor server instances");
```

Comment:
If you want a shorter flag name, go with ip_server*. But I'll rather have include_processor_server*. The convention is to use readable names for flags.

Reply:
Done.